### PR TITLE
Cbapi-2988: Resolve: EnrichedEvent search Error: TypeError: TimeoutError does not take keyword arguments

### PR DESCRIPTION
**Pull request by abowersox-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Tests have been added that prove the fix is effective or that the feature works.
- [x] New and existing tests pass locally with the changes.
- [x] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [x] A self-review of the code has been done.

## What is the ticket or issue number?

https://jira.carbonblack.local/browse/CBAPI-2988

## Pull Request Description

Fixed imports in endpoint_standard/base.py, cbc_sdk/base.py, and platform/alerts.py to import the SDK TimeoutError rather than vanilla TimeoutError (which does not take keyword arguments).  Also added tests to make sure our TimeoutError is thrown properly where it ought to be.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## How Has This Been Tested?

New unit tests have been added to make sure our TimeoutError is thrown properly where it ought to be.  Unit test has been tested by removing the new import and seeing that new test errors out.

### CODE REVIEW

**Review by avanbrunt-cb — APPROVED** [review]

**Review by lhilmes-cb — APPROVED** [review]
I'm probably not the best one to review this, but I want to unblock if you're waiting on a second approval.

**Pull request merged**
